apps/src/code-studio/components/AddAssetButtonRow.jsx

```diff
@@ -51,6 +53,7 @@ export default class AddAssetButtonRow extends React.Component {
     onUploadError: PropTypes.func.isRequired,
     onSelectRecord: PropTypes.func.isRequired,
     statusMessage: PropTypes.string,
+    recordDisabled: PropTypes.bool,
     // Temporary prop until recording audio is widely released
     recordEnabled: PropTypes.bool
   };
```

Comment:
In this context `recordEnabled` means that recording is a possibility available to those with the flag on? And `recordDisabled` means recording is possible, but temporarily disabled because a recording is already in progress?   If this is the case, given the comment that `recordEnabled` is temporary, that seems fine; otherwise, it's confusing to have two opposite boolean props.  Could mitigate with a rename to something like `recordExperimentEnabled`.

Reply:
Yes, recordEnabled in this case just means that the flag is on. Poorva is in a rush to get this feature released so we'll hopefully pull this prop out in the next few days. For that reason, I'm tempted to consider this not ship-blocking and apply better naming in the future.